committeeoversightapp/static/css/custom.css

```diff
@@ -20,6 +20,14 @@ body {
   background-color: var(--green-color);
 }
 
+.btn-danger {
+  background-color: var(--red-color);
+}
+
+.confirm-delete {
+
+}
```

Comment:
Looks like you're missing something here?

Reply:
Oops!